components/brave_rewards/resources/request_ads_enabled/lib/host.ts

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Host, HostState } from './interfaces'
```

Comment:
why is this named `Host`? If it's meant to match `BraveAdsHost` that messes up the terminology. `Host` is for the browser process and this is running in the renderer.

Reply:
spoke to @zenparsing and agreed using `Client` terminology here (and other places in rewards) makes more sense with the chromium conventions